internal/kafka/command_cluster_describe.go

```diff
@@ -115,7 +116,7 @@ func (c *clusterCommand) outputKafkaClusterDescription(cmd *cobra.Command, clust
 	return table.Print()
 }
 
-func convertClusterToDescribeStruct(cluster *cmkv2.CmkV2Cluster, ctx *config.Context) *describeStruct {
+func convertClusterToDescribeStruct(cluster *cmkv2.CmkV2Cluster, ctx *config.Context, client *ccloudv2.Client) *describeStruct {
```

Comment:
Make the API call beforehand, and pass the network ID instead of the client? It should be much easier to unit test this way!